tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -48,25 +48,6 @@ SPECTRE_TEST_CASE("Unit.DataStructures.DataVector", "[DataStructures][Unit]") {
   CHECK(t_move_constructor.is_owning());
 }
 
-SPECTRE_TEST_CASE("Unit.Serialization.DataVector",
```

Comment:
Why move this at all? I usually just use search to find the test I need to edit.